packages/frontend/src/components/MenuBar/MenuItem.tsx

```diff
@@ -0,0 +1,109 @@
+import { ChevronRightIcon, ExternalLinkIcon } from '@navikt/aksel-icons';
+import cx from 'classnames';
+import styles from './menuItem.module.css';
+
+interface DropDownMenuItemProps {
+  displayText?: string;
+  label?: string;
+  icon?: React.ReactNode;
+  path?: string;
+  onClick?: () => void;
+  count?: number;
+  onClose?: () => void;
+  isExternalLink?: boolean;
+  isActive?: boolean;
+  isInbox?: boolean;
+  isWhiteBackground?: boolean;
+  leftContent?: React.ReactNode;
+  rightContent?: React.ReactNode;
+  fullWidth?: boolean;
+  smallIcon?: boolean;
+  smallText?: boolean;
+}
+
+export const MenuItem = ({
+  displayText,
+  label,
+  icon,
+  path,
+  onClick,
+  count,
+  onClose,
+  isExternalLink,
+  isActive,
+  isInbox,
+  isWhiteBackground,
+  leftContent,
+  rightContent,
+  fullWidth,
+  smallIcon,
+  smallText,
+}: DropDownMenuItemProps) => {
+  const renderDropDownMenuItem = () => (
```

Comment:
Drag this out as its own component instead? You can just keep it as an unexported component in the same file.